docs/pages/guides/typescript.mdx

```diff
@@ -275,12 +275,8 @@ module.exports = config;
 
 **app.config.ts** is supported by default. However, it doesn't support external TypeScript modules, or **tsconfig.json** customization. You can use the following approach to get a more comprehensive TypeScript setup:
 
-```js app.config.js
-require('ts-node/register');
-module.exports = require('./app.config.ts');
-```
-
 ```ts app.config.ts
+import 'ts-node/register' // Add this to import TypeScript files
```

Comment:
is this import required? i don't use it here: https://github.com/brentvatne/microfoam-app/blob/main/apps/mobile/app.config.ts